libethereum/Utility.cpp

```
@@ -126,9 +126,9 @@ void dev::eth::upgradeDatabase(std::string const& _basePath)
 				}
 			}
 		}
-		catch (...)
+		catch (boost::exception const& _e)
```

You will not catch all exceptions any more.